### PR TITLE
HW7 Andrii Voinenko

### PR DESCRIPTION
**Pull request by andrii-voinenko**

### CODE REVIEW

File: Andrii_Voinenko/Lesson7/task2.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import math
+
+
+def square(square_side_value: float) -> tuple:
+    calc_perimeter = square_side_value * 4
+    calc_square_area = pow(square_side_value, 2)
+    calc_diagonal = square_side_value * math.sqrt(2)
+    result = (calc_perimeter, calc_square_area, round(calc_diagonal, 2))
+    return print(result)
```
Comment (bohdandovbsyh) [inline Andrii_Voinenko/Lesson7/task2.py]: return result 

print('smth') returns None

File: Andrii_Voinenko/Lesson7/task3.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
+def is_prime(number_to_check: int) -> bool:
+    if number_to_check > 1 and number_to_check < 1001:
+        for num in range(2, number_to_check):
+            if (number_to_check % num) == 0:
+                return False
+        return True
+    else:
+        return print(f"Enter number more then 2 and less than 1000")
```
Comment (bohdandovbsyh) [inline Andrii_Voinenko/Lesson7/task3.py]: no printing in return state

File: Andrii_Voinenko/Lesson7/task4.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+        # Checking if char != digit
+        for char in text:
+            if not char.isdigit():
+                without_digits.append(char)
+
+        # Adding all chars without digits
+        result = ''.join(without_digits)
+
+    return print(result)
```
Comment (bohdandovbsyh) [inline Andrii_Voinenko/Lesson7/task4.py]: do not use print in return state

**Comment by andrii-voinenko** [comment]
I have removed print from return in functions

**Review by bohdandovbsyh — APPROVED** [review]

**Pull request merged**